experimental/packages/opentelemetry-instrumentation-http/test/utils/assertSpan.ts

```diff
@@ -65,7 +66,19 @@ export const assertSpan = (
   );
 
   assert.strictEqual(span.links.length, 0);
-  assert.strictEqual(span.events.length, 0);
+
+  if (validations.error) {
+    assert.strictEqual(span.events.length, 1);
+    assert.strictEqual(span.events[0].name, 'exception');
+
+    const eventAttributes = span.events[0].attributes as Object;
```

Comment:
I'd suggest asserting the `eventAttributes` to be non-nullish rather than casting it as an object -- the casting has no runtime-effect, we'll get a TypeError below if it is actually an `undefined`.
```suggestion
    const eventAttributes = span.events[0].attributes;
    assert.ok(eventAttributes != null);
```